components/HistoryTable.qml

```diff
@@ -443,7 +443,7 @@ ListView {
         id: dropModel
         ListElement { name: "<b>Copy address to clipboard</b>"; icon: "../images/dropdownCopy.png" }
         ListElement { name: "<b>Add to address book</b>"; icon: "../images/dropdownAdd.png" }
-        ListElement { name: "<b>Send to same destination</b>"; icon: "../images/dropdownSend.png" }
+        ListElement { name: "<b>Send to this address</b>"; icon: "../images/dropdownSend.png" }
```

Comment:
FYI: This is currently unused as the code above for `TableDropdown` is commented out.